### DISCUSSION TITLE
Gamma issues.

### DISCUSSION THREAD

**Discussion by PrinceMinewalker** [PrismLauncher/PrismLauncher#3242] [Q&A] [answered]
I've been trying to edit my gamma in a specific instance, but every time I load the game in that instance using PrismLauncher the options.txt file resets all changes back to the default setting. Does anybody know what could be the issue?

**Comment by maskersss** [comment]
newer mc versions don't allow you to set a gamma value above 1.0, you should use a mod instead

**Reply by PrinceMinewalker** [reply]
This is in version 1.16.1 though.

**Reply by PrinceMinewalker** [reply, marked as answer]
I managed to fix it. It turns out I didn't realize that I had a different launcher running in the background.
